Companies/update (#115)
MewsSystems/gitbook-connector-api: operations/enterprises.md

    "TaxIdentifier": null,
    "AdditionalTaxIdentifier": null,
    "BillingCode": null,
    "AccountingCode": null,
    "InvoiceDueInterval": {
        "Value": "P2DT23H"
    },
    "ContactPerson": {

>         "Value": "SamplePerson"

Reviewer: ```suggestion
        "Value": "John Snow"
```

    },
    "Contact": {

>         "Value": "ContactInfo"

Reviewer: ```suggestion
        "Value": "John Snow"
```

| `AdditionalTaxIdentifier` | [String update value](reservations.md#string-update-value) | optional | Additional tax identifer of the company \(or `null` if the additional tax identifier should not be updated\). |
| `BillingCode` | [String update value](reservations.md#string-update-value) | optional | Billing code of the company \(or `null` if the billing code should not be updated\). |
| `AccountingCode` | [String update value](reservations.md#string-update-value) | optional | Accounting code of the company \(or `null` if the acounting code should not be updated\). |
| `Options` | Array of [Company option](enterprises.md#company-options)  | optional | Options of the company. |

> | `InviceDueInterval` | string | optional | Start of the interval in UTC timezone in ISO 8601 format. |

Reviewer 2: Shouldn't be these included in sample request as well?

| `AdditionalTaxIdentifier` | [String update value](reservations.md#string-update-value) | optional | Additional tax identifer of the company \(or `null` if the additional tax identifier should not be updated\). |
| `BillingCode` | [String update value](reservations.md#string-update-value) | optional | Billing code of the company \(or `null` if the billing code should not be updated\). |
| `AccountingCode` | [String update value](reservations.md#string-update-value) | optional | Accounting code of the company \(or `null` if the acounting code should not be updated\). |

> | `InviceDueInterval` | string | optional | Start of the interval in UTC timezone in ISO 8601 format. |

Reviewer: Typo

> | `InviceDueInterval` | string | optional | Start of the interval in UTC timezone in ISO 8601 format. |

Reviewer: This is not a start of the interval with Timezone!

> | `InviceDueInterval` | string | optional | Start of the interval in UTC timezone in ISO 8601 format. |

Reviewer: ```suggestion
| `InvoiceDueInterval` | string | optional | The maximum time, when the invoice has to be be paid in ISO 8601 duration format. |
```

> | `InvoiceDueInterval` | string | optional | The maximum time, when the invoice has to be be paid in ISO 8601 duration format. |

Reviewer 3: Those do not seem to be strings but update values of type string. Please update. CC @vitasamek

> | `Contact Person` | string | optional | Contact Person of the company. |

Reviewer: How come there is a space? The name of the attribute is without space, always!

> | `Contact Person` | string | optional | Contact Person of the company. |

Reviewer: ```suggestion
| `ContactPerson` | string | optional | Contact person of the company. |
```
